packages/snaps-execution-environments/src/common/utils.ts

```diff
@@ -174,3 +186,22 @@ export function sanitizeRequestArguments(value: unknown): RequestArguments {
   const json = JSON.parse(JSON.stringify(value));
   return getSafeJson(json) as RequestArguments;
 }
+
+/**
+ * Check if the input is a valid response.
+ *
+ * @param response - The response.
+ * @returns True if the response is valid, otherwise false.
+ */
+export function isValidResponse(response: Record<string, unknown>) {
```

Comment:
You have a test for this function, but codecov is saying these lines are uncovered? 🤔